plugins/filestorage/localfs/index.js

```diff
@@ -290,6 +291,57 @@ LocalFileStorage.prototype.getFileStats = function (filePath, callback) {
 };
 
 /**
+ * Copies an asset from one tenant to another.
+ * 
+ * @param {object} asset - A valid 'asset' record
+ * @param {string} sourceTenantName - Source tenant.name
+ * @param {string} destinationTenantName - Destination tenant.name
+ */
+LocalFileStorage.prototype.copyAsset = function(asset, sourceTenantName, destinationTenantName, callback) {
+  // Verify the destination folder exists
+  var assetFolder = path.join(configuration.serverRoot, configuration.getConfig('dataRoot'), destinationTenantName, asset.directory);
```

Comment:
You could make this a bit more readable (with shorter lines) by declaring
`var dataRoot = path.join(configuration.serverRoot, configuration.getConfig('dataRoot');`

There are 5 instances it could be used.